app/directives/remark/remark.directive.html

```diff
@@ -8,17 +8,19 @@
                         <h4 class="rmk-header-text" translate translate-cloak>STR_OWN_REMARK</h4>
                     </div>
                     <div class="col-xs-3 col-sm-3 col-md-2 col-lg-1">
-                        <div class="rmk-header-button">
-                            <button class="btn btn-primary btn-sm" ng-click="c.clicked(true)" ng-disabled="c.loading" uib-tooltip=""><span class="glyphicon glyphicon-ok"></span></button>
-                            <button class="btn btn-warning btn-sm" ng-click="c.clicked(false)" ng-disabled="c.loading" uib-tooltip=""><span class="glyphicon glyphicon-remove"></span></button>
+                        <div class="rmk-header-button btn-group">
+                            <button class="btn btn-primary btn-sm" ng-click="c.clicked(true)" ng-disabled="c.loading || !c.isUnsaved" uib-tooltip="{{'STR_SAVE' | translate}}"
+                                tooltip-placement="bottom"><span class="glyphicon glyphicon-ok"></span></button>
+                            <button class="btn btn-warning btn-sm" ng-click="c.clicked(false)" ng-disabled="c.loading || !c.isUnsaved" uib-tooltip="{{'STR_CANCEL' | translate}}"
```

Comment:
Should this use a new `db-btn-warn` similar to `db-btn-dang` but with different background-color?